eng/dependabot/Packages.props

```diff
@@ -19,7 +19,7 @@
     <PackageVersion Include="LargeAddressAware" Version="1.0.5" />
     <PackageVersion Update="LargeAddressAware" Condition="'$(LargeAddressAwareVersion)' != ''" Version="$(LargeAddressAwareVersion)" />
 
-    <PackageVersion Include="Microsoft.BuildXL.Processes" Version="0.1.0-20230929.2" />
```

Comment:
I don't think we should take this in servicing. Can we mirror the older package instead?

Reply:
This version of the package came from a deleted BuildXL feed.
I looked into it and what currently happens on the branch is that neither of these versions are in any available feed so it restores the latest Microsoft.BuildXL.Processes 0.1.0-20240606.3 from https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json 

Who can I contact to add the old BuildXL 0.1.0-20230929.2 to the feed? I don't know where it comes from and who's responsible for the feed. Alternative solution: we could update this branch to point to 20240606.3

Also I noticed that Microsoft.BuildXL.Processes is restored from https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json while Microsoft.BuildXL.Utilities.Core and Microsoft.BuildXL.Native from https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json
shouldn't they both come from the same feed?

Reply:
@JanProvaznik I'll add you to an internal thread on this.